Stronger/Resources/Localizable.xcstrings

```diff
@@ -504,7 +507,29 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Spezi Modules"
+            "value" : "Aerobics tracker"
+          }
+        }
+      }
+    },
+    "WELCOME_AREA4_DESCRIPTION" : {
+      "extractionState" : "manual",
```

Comment:
Please try to avoid the `"extractionState" : "manual"` flag within the String catalog. During the build step, Xcode automatically extracts all used Strings in the application and puts them into the String Catalog. That helps a lot with keeping the String Catalog in sync with the underlying Views.